src/main/java/learn/petgallery/models/EntityEvent.java

```diff
@@ -0,0 +1,48 @@
+package learn.petgallery.models;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Data
+@NoArgsConstructor
```

Comment:
This can be removed, just leaving it open while I played with constructor vs factory